### PR TITLE
Page titles, urls, descriptions

### PR DESCRIPTION
**Pull request by ZooeyMiller**
We've added initial unique titles and descriptions to each html page, my intention with them was that they were to be iterated upon and updated with perhaps more streamlined copy, but for now at least they are all unique which is important for SEO. 

One of the URLs/filenames has also been changed. 

#344 #307 #348

### CODE REVIEW

**Comment by finnhodgkin** [comment]
Not many of the WiP titles include keywords. The Moz guide suggests that adding a few _relevant_ target words to the start of a title can help to influence search ranking. 

`dwyl - our blog` vs `dwyl - web development blog` 

There's also a decision to be made about the location of 'dwyl' in titles. IMO because the url is 'dwyl.com', it's a bit of a waste of prime SEO real estate to put the brand at the start. It also makes for more readable tabs / titles if the purpose of the page is displayed first:

'Web development blog - dwyl'

File: redesign/blog.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="This is our blog, where we talk about our experiences as an agile software development agency">
```
Comment (markwilliamfirth) [inline redesign/blog.html]: The dwyl blog, where you can read about digital product development, agile methodologies, open source and much more

File: redesign/blog.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Web development blog - dwyl</title>
```
Comment (markwilliamfirth) [inline redesign/blog.html]: dwyl | The Blog | Web Development, Agile Methodologies and more

File: redesign/blog.html [hunk 3]
```diff
@@ -35,15 +35,15 @@
...
+            <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl heart logo" class="w1-plus"/>
             dwyl
           </a>
         </li>
         <li class="ph5 pt5 pb2 tl dn-ns"><a href="/dwyl-site/" class="white link">Home</a></li>
         <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/dwyl-site/portfolio" class="white link">Portfolio</a></li>
         <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/dwyl-site/values" class="white link">Values</a></li>
         <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/dwyl-site/team" class="white link">Team</a></li>
-        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/dwyl-site/app" class="white link">App
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/dwyl-site/time-app" class="white link">App
```
Comment (markwilliamfirth) [inline redesign/blog.html]: great

File: redesign/careers.html [hunk 4]
```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Meet our core team of developers, designers, scrum masters and more.">
```
Comment (markwilliamfirth) [inline redesign/careers.html]: this is the careers page, so perhaps:

Learn what it's like to be a developer, designer or scrum master at our open source product development agency

File: redesign/case-studies/ellen-macarthur-foundation/CE100.html [hunk 5]
```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Find out more about the online membership platform app that we built for the Ellen Macarthur Foundation.">
```
Comment (markwilliamfirth) [inline redesign/case-studies/ellen-macarthur-foundation/CE100.html]: great

File: redesign/case-studies/ellen-macarthur-foundation/CE100.html [hunk 6]
```diff
@@ -20,7 +20,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Case study - The Ellen MacArthur Foundation - dwyl</title>
```
Comment (markwilliamfirth) [inline redesign/case-studies/ellen-macarthur-foundation/CE100.html]: dwyl | Case Study | The Ellen MacArthur Foundation

File: redesign/case-studies/inclusive-classrooms/skills-wheel.html [hunk 7]
```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Find out more about the data collection and visualistion app that we built for Inclusive Classroom to help them help young people">
```
Comment (markwilliamfirth) [inline redesign/case-studies/inclusive-classrooms/skills-wheel.html]: Find out more about the data collection and visualisation education app that we built for Inclusive Classrooms

File: redesign/case-studies/science-museum-group/online-collections.html [hunk 8]
```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Find out more about the Online Databank we made for the Science Museum Group, so that all of the SMG's data and artefacts could be accessible to the public">
```
Comment (markwilliamfirth) [inline redesign/case-studies/science-museum-group/online-collections.html]: Find out more about the Online Databank app we built for the Science Museum Group - a public and accessible register for all of their artefacts and associated data

File: redesign/community.html [hunk 9]
```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems, find out more about them here!">
```
Comment (markwilliamfirth) [inline redesign/community.html]: *find out more about the dwyl community here

File: redesign/community.html [hunk 10]
```diff
@@ -20,7 +20,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Join our community of open source developers! - dwyl</title>
```
Comment (markwilliamfirth) [inline redesign/community.html]: dwyl | Our Community | Join Our Community of Open Source Developers

File: redesign/index.html [hunk 11]
```diff
@@ -20,7 +20,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Agile development agency - dwyl</title>
```
Comment (markwilliamfirth) [inline redesign/index.html]: dwyl | App, Website and Software Development | London

File: redesign/portfolio.html [hunk 12]
```diff
@@ -20,7 +20,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Portfolio of web and mobile apps - dwyl</title>
```
Comment (markwilliamfirth) [inline redesign/portfolio.html]: dwyl | Portfolio | Responsive Web and Mobile Applications

File: redesign/team.html [hunk 13]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Find out more about our core product development team at dwyl, based in East London, making your dreams come true">
```
Comment (markwilliamfirth) [inline redesign/team.html]: Find out more about our core product development team at dwyl, based in East London, using technology to solve real problems

File: redesign/team.html [hunk 14]
```diff
@@ -13,7 +13,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Product development team - dwyl</title>
```
Comment (markwilliamfirth) [inline redesign/team.html]: dwyl | The Team | Product Developers, UX/UI Designers, Scrum Masters and more

File: redesign/time-app.html [hunk 15]
```diff
@@ -13,7 +13,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Time app - become a beta tester for our time management tool - dwyl</title>
```
Comment (markwilliamfirth) [inline redesign/time-app.html]: dwyl | Time | Become a Beta Tester for Our Time Management App

File: redesign/values.html [hunk 16]
```diff
@@ -13,7 +13,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>What we believe in - dwyl</title>
```
Comment (markwilliamfirth) [inline redesign/values.html]: dwyl | Our Values | People, Plants and the Planet

File: redesign/case-studies/science-museum-group/online-collections.html (hunk 8 above)
Comment (iteles) [reply redesign/case-studies/science-museum-group/online-collections.html]: Online Collections (it's not an 'app' per se)

File: redesign/index.html (hunk 11 above)
Comment (iteles) [reply redesign/index.html]: [...] Development & Training

File: redesign/blog.html (hunk 2 above)
Comment (iteles) [reply redesign/blog.html]: @markwilliamfirth Did you see @finnhodgkin's comment above? https://github.com/dwyl/dwyl-site/pull/347#issuecomment-330883526

**Comment by ZooeyMiller** [comment]
Suggested changes have been made, with dwyl put to the end of titles.

**Pull request merged**